src/Features/VisualBasic/Portable/VBFeaturesResources.resx

```diff
@@ -1247,4 +1247,7 @@ Sub(&lt;parameterList&gt;) &lt;statement&gt;</value>
   <data name="Use_IsNot_Nothing_check" xml:space="preserve">
     <value>Use 'IsNot Nothing' check</value>
   </data>
+  <data name="Warning_code_may_overflow_at_runtime_unless_removeintchecks_flag_is_supplied_to_compiler" xml:space="preserve">
+    <value>Warning: code may overflow ar runtime unless /removeintchecks flag is passed to compiler</value>
```

Comment:
Typo: overflow ar runtime

You basically say "We are only able to provide an algorithm that fails under non deterministic occasions at runtime until you remove a safety net ( which many developers are not allowed to remove). Good luck with that.". I would say that anything, even the most terrible algorithm, is better than obviously buggy code.

Reply:
> I would say that anything, even the most terrible algorithm, is better than obviously buggy code.

As i mentioned in the other thread, i'm happy to take an algorithm that will work for both cases.  That said, in the interim perioud, this at least lets the user know about the issue.

Reply:
You are right. This feature is shipped and needs to be fixed soon. I posted some other ideas in the corresponding issue. Based on those we can make a new PR if we are confident to have a better solution.